### ISSUE TITLE
Correct similar tokens (> 1000)

### ISSUE THREAD

**Issue by paulo170** [hipster-philology/pyrrha#176]
Hi,

I am working on a huge dataset (~600.000 tokens), and I would like to massively edit similar tokens. Some of them are very common (more than 20.000).
When I am trying to apply a modification to more than 1000 similar tokens, only the first 1000 tokens are edited. Is there a way to increase this limitation?

I have updated the following line
```python
tokens = WordToken.get_similar_to_record(change_record=record).paginate(per_page=100000)
```
in tokens.py but of course my browser crash because of the number of lines to load in a page.

Any idea?
Thanks!

**Comment by PonteIneptique** [comment]
Hmm, that's a tricky question. But indeed, 100k tokens on a single page looks like something a browser would not like. We also have an issue when you apply the same thing to too much tokens ( https://github.com/hipster-philology/pyrrha/issues/166 ).
Can you try 10k instead of 100k ? 

If you are familiar with python, there is a simple fix for #166 which is to iterate over the data instead of treating everything at once.

**Comment by paulo170** [comment]
Indeed, my browser accept with 10k, it's a good compromise, thx (it takes a lot of time, obviously, around 4-5 minutes for 6500 tokens).
Concerning the other issue, I am not able to reproduce it but I see the idea!
